src/workerd/util/asan-options.h

```diff
@@ -0,0 +1,7 @@
+#include <kj/string.h>
```

Comment:
no headers required here

Reply:
Thanks, I made a copy of the TLS config util file, which used kj::StringPtr